### DISCUSSION TITLE
[v5] Routing with domains - TLD wildcard possible?

### DISCUSSION THREAD

**Discussion by e-e** [adonisjs/core#3394] [Help] [answered]
I would like to do something like:

```typescript
Route.group(() => {
  // ...
}).domain("domainA.example.*");

Route.group(() => {
  // ...
}).domain("domainB.example.*");
```

So that I can set up my local hosts file as `domainA.example.local` and `domainB.example.local` and in production use `domainA.example.com`, etc.

Is something like this possible? I have tried it as show above and that doesn't seem to be working - is there another way? If not, what is the recommended way to use domains in local development?

**Comment by tomgobich** [comment, marked as answer]
There might be a better way, but in the past I’ve used environment variables for this. 

```typescript
import Env from “@ioc:Adonis/Core/Env”

Route.group(() => {
  // …
}).domain(`domainA.${Env.get(“ENV_NAME”)}`)
```
(on phone apologies if this doesn’t come through with proper formatting)

**Reply by e-e** [reply]
Thanks. I was hoping there was another option but this will probably be fine
